presto-main/src/main/java/com/facebook/presto/server/InternalCommunicationConfig.java

```diff
@@ -15,13 +15,20 @@
 
 import io.airlift.configuration.Config;
 
+import java.io.File;
+
 public class InternalCommunicationConfig
 {
     private boolean httpsRequired;
     private String keyStorePath;
     private String keyStorePassword;
     private String ldapUser;
     private String ldapPassword;
+    private String kerberosPrincipal;
+    private String kerberosServiceName;
+    private boolean kerberosEnabled;
```

Comment:
ok, and instead what do you think we should take as a flag to enable kerberos auth between nodes? I was trying to replicate the CLI side params here. For LDAP, we use the "user" as the flag.